plugins/scpi.c

```diff
@@ -309,7 +309,6 @@ static int tty_read(struct scpi_instrument *scpi)
 	clock_gettime(CLOCK_MONOTONIC, &ts_current);
 	nsecs = ts_current.tv_nsec + (SOCKETS_TIMEOUT * pow(10.0, 9));
 	ts_end.tv_sec = ts_current.tv_sec + (nsecs / pow(10.0, 9));
-	ts_end.tv_nsec = nsecs % (unsigned long long) pow(10.0, 9);
 	ts_end.tv_nsec = ts_current.tv_nsec;
```

Comment:
But does that make sense? Shouldn't the second line not rather be `+=` instead of `=`?

Reply:
I'm not sure to be honest.
I'll have to check

Reply:
yep, good point ; it makes sense to switch to `+=`
thanks for catching this